Maddori.Apple/Maddori.Apple/Screen/Home/Home/HomeViewController.swift

```diff
@@ -258,8 +275,12 @@ final class HomeViewController: BaseViewController {
         ).responseDecodable(of: BaseModel<CertainTeamDetailResponse>.self) { json in
             if let json = json.value {
                 guard let teamName = json.detail?.teamName else { return }
+                guard let isAdmin = json.detail?.admin else { return }
```

Comment:
p5;

요기도 하나의 guard let 으로 합쳐주세용.. 하하
```suggestion
                guard let teamName = json.detail?.teamName,
                        let isAdmin = json.detail?.admin
                        else { return }
```

Reply:
아 분명히 했는데~ 왜 안했냐고~
[07dec45](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/142/commits/07dec459952e22a7b5fa06f3687d733b8a8ba657)